src/PerformanceTests/Tests/Statistics.cs

```diff
@@ -0,0 +1,130 @@
+﻿using System;
```

Comment:
Also, you didn't move / rename the file as it is mared as "Added" in git, loosing history on its change